pkgs/jnigen/lib/src/generate_bindings.dart

```diff
@@ -45,7 +44,6 @@ Future<void> generateJniBindings(Config config) async {
   classes.accept(Excluder(config));
```

Comment:
Add a note here to keep in sync with `GenerationStage`. Better yet, write a wrapper closure around `classes.accept` that captures a stage variable:

```Dart
GenerationStage stage = GenerationStage.unprocessed;
void runStage(GenerationStage newStage, Visitor visitor) {
  assert(newStage > stage);
  stage = newStage;
  classes.accept(visitor);
}
```

Reply:
Good idea! Done